server/src/main/clients/web-browser/src/ProjectDashboard/ProjectDashboard.tsx

```diff
@@ -0,0 +1,11 @@
+import './ProjectDashboard.scss';
+
+function ProjectDashboard() {
+  return (
+    <section>
+      <h2>Post Tracking</h2>
+    </section>
+  );
+}
+
+export {ProjectDashboard};
```

Comment:
ProjectDashboard should be in src/pages/projects/ProjectDashboard

Reply:
This isn't actually its own page, it is part of the Overview page, so I didn't put it in pages

Reply:
Ok, let's put them in libs then. I'm trying to start organizing the files better than having them all at the top level. Maybe libs/projects?

Reply:
Moved